addons/account/static/src/js/reconciliation/reconciliation_model.js

```diff
@@ -972,47 +966,45 @@ var StatementModel = BasicModel.extend({
             var line = _.find(self.lines, function (l) {
                 return l.id === data.st_line.id;
             });
+            // Line has been already reconciled by the reconciliation model.
+            if(data.status === 'reconciled'){
```

Comment:
I would change the python instead and only return the non reconciled entries, and add a parameter `reconciled_aml_ids` that returns the list of all aml_ids that have been automatically reconciled. That way it is even easier to display the message if there is automatically reconciled entries, just change the xml view and add a t-if="state.reconciled_aml_ids &amp;&amp; state.reconciled_aml_ids.length" that will display the notification message. (that way we can also remove most of the code added in the ´start´ function of the reconciliation_action.js